RecoEgamma/EgammaTools/python/egammaGainSwitchFlag_cfi.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+PhotonGainSwitchFlagProducer = cms.EDProducer('PhotonGainSwitchFlagProducer',
```

Comment:
cfi files should really have just one instance of a producer defined.
The contents of this file can just be fully moved to egammaGainSwitchFlag_cff
without the extra indirection